src/app/components/application/personal-info/parent/parent.component.html

```diff
@@ -1,6 +1,6 @@
 <h3 class="mb-4">Parent or Guardian</h3>
-
-<div class="nestedForm">
+<div *ngIf="isLoading == true" ><i class="fa fa-spinner fa-fw fa-pulse"></i> Processing</div>
```

Comment:
this should use 3 equal signs to enforce strict equality.

You can read more about it here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness